src/index.js

```diff
@@ -129,7 +130,7 @@ class Node extends EventEmitter {
         this.switch.transport.add(
           transport.tag || transport.constructor.name, transport)
       } else if (transport.constructor &&
-                 transport.constructor.name === 'WebSockets') {
+                 transport instanceof WebSockets) {
```

Comment:
Same feedback as https://github.com/multiformats/js-multiaddr/pull/55#discussion_r173337049

Reply:
@diasdavid hey, 

**solution 1** : What if we add a `name` property to the `WebSockets` class which isn't gonna be mangled by uglify like so
```javascript
class WebSockets {
 constructor (opts) {
   this.name = 'WebSockets'
 }
}
```

**Solution 2** : use both `instanceOf` and `constructor,name` like so
` if (transport.constructor.name === 'WebSockets' || transport instanceOf WebSockets)`

What do you think.??

Reply:
solution 1 is closer to a real solution. Even better would be just to have a `isWebSockets` method